server/routes/routes.js

```diff
@@ -27,10 +35,21 @@ const router = express.Router();
 // 	}
 // });
 
+router.use(passport.initialize());
+router.use(passport.session());
+
 router.get('/', (req, res) => {
-	res.sendFile('index.html', {
-		root: './dist'
-	});
+	if (req.user) {
```

Comment:
Fine for debugging, but clean this up after. We can do better with something like
```
if (req.user) {
logger.info
} else {
logger.info
}

res.sendfile
```

or simply be smarter about your logs, where you exploit a ternary, or other boolean expression

Reply:
Purely for debugging and can be removed once ready